src/components/AccessibilityGuide/SkipNavigation.stories.mdx

```diff
@@ -20,8 +20,6 @@ the top of the page and is visually hidden until a user focuses on the link.
 In the case of NYPL applications, the skip navigation contains two links; the
 first link points to the main content of the page and the second link points to
 accessibility information on [NYPL.org/accessibility](https://nypl.org/accessibility).
-For most digitial applications on the NYPL.org platform, the [NYPL Header](https://github.com/NYPL/dgx-header-component)
```

Comment:
Can this refrence be kept, but reworded to refer to the new app that Jackie is working on?